functions/functions.go

```diff
@@ -0,0 +1,24 @@
+package functions
+
+import "embed"
+
+//go:embed *.sql
+var functions embed.FS
+
+var Functions map[string]string
+
+func GetFunctions() (map[string]string, error) {
```

Comment:
How is this being called ?

Reply:
https://github.com/flanksource/incident-commander/pull/218/files#diff-953a885e28970388a5ee8146da2db0488f6c2539fe2db2caca45a4033f090985R138